packages/symfony-static-dumper/src/Command/DumpStaticSiteCommand.php

```diff
@@ -21,26 +22,40 @@ public function __construct(
         ParameterBagInterface $parameterBag
     ) {
         parent::__construct();
-
-        $projectDir = (string) $parameterBag->get('kernel.project_dir');
-
-        $this->publicDirectory = $projectDir . '/public';
-        $this->outputDirectory = getcwd() . '/output';
     }
 
     protected function configure(): void
     {
         $this->setName('dump-static-site');
         $this->setDescription('Dump website to static HTML and CSS in the output directory');
+
+        // Adding arguments options for the main command
+        $this->addOption('public-dir', 'p', InputOption::VALUE_REQUIRED, 'Define the input public directory relative to the root project directory');
+        $this->addOption('output-dir', 'o', InputOption::VALUE_REQUIRED, 'Define the output directory relative to the execution of the comand');
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $this->publicDirectory = $this->getPublicDirectory($input->getOption('public-dir'));
+        $this->outputDirectory = $this->getOutputDirectory($input->getOption('output-dir'));
+        
         $this->symfonyStyle->section('Dumping static website');
         $this->symfonyStaticDumperApplication->run($this->publicDirectory, $this->outputDirectory);
```

Comment:
This should re-use the variable here:

```suggestion
        $publicDirectory = $this->getPublicDirectory($input->getOption('public-dir'));
        $outputDirectory = $this->getOutputDirectory($input->getOption('output-dir'));
        
        $this->symfonyStyle->section('Dumping static website');
        $this->symfonyStaticDumperApplication->run($publicDirectory, $outputDirectory);
```